docs/source/tutorials/first_kagome_chain.md

```diff
@@ -24,26 +24,26 @@ In this tutorial you will learn how to execute Kagome-based Polkadot-host chain
 
 ### Tutorial
 
-During this tutorial we will:
-1. Launch Kagome network using prepared genesis file
-2. Query the balance of Alice account
-3. Send extrinsic that transfers some amount of currency from Alice to Bob
-4. Query again the balance of Alice to make sure balance was updated
+In this tutorial, we will:
+1. Launch the Kagome network using a prepared genesis file.
+2. Query the balance of Alice's account.
+3. Send an extrinsic that transfers a specific amount of currency from Alice to Bob.
+4. Query Alice's balance again to confirm that it has been updated.
 
 #### Launch Kagome network
 
-For this tutorial we will spin up a simple network of a single peer with predefined genesis.
+In this tutorial, we will setup a basic network consisting of a single peer with a predefined genesis.
 
-To start with let's navigate into the node's folder:
+Firstly, we need to move to the node's directory:
 
 ```shell script
 cd examples/first_kagome_chain
 ```
 
 `first_kagome_chain` folder contains necessary configuration files for our tutorial:
 
-* `localchain.json` – genesis file for our network. It contains necessary key-value pairs that should be inserted before the genesis block
-* `base_path` – Directory, containing kagome base path. It contains several dirs, each one named with the chain id, which data it stores (`dev` in this case). Data for each chain consists of `db/` (will be initialized on node startup) and `keystore/` (keys to sign the messages sent by our authority). The latter has to exist prior to the node start. This behaviour will be improved in the future.
+* `localchain.json` – This is the genesis file for our network. It houses the crucial key-value pairs that need to be inserted before the creation of the genesis block.
+* `base_path` – This directory encompasses the base path of the Kagome project. Inside, you'll find the 'chains' folder, which contains several directories. Each of these directories carries the name of the chain id it stores the data for (`rococo-dev` in this case). The data for each chain is comprised of `db/` (which gets initialized upon node startup) and `keystore/` (which contains the keys used to sign messages sent by our authority). It's important to note that the `keystore/` must exist before the node starts. However, there's an exception for predefined accounts such as Alice, Bob, etc.
```

Comment:
<strike>keystore must exist before</strike> (keystore directory not required, keys can be added later)